src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -454,6 +462,23 @@ class OSRow extends React.Component {
     }
 }
 
+const OfflineTokenRow = ({ offlineToken, onValueChanged, validationFailed }) => {
+    const validationStateToken = validationFailed.offlineToken ? 'error' : 'default';
+
+    return (
+        <FormGroup label={_("Offline Token")} fieldId="offline-token"
+                   id="offline-token-group"
+                   helperTextInvalid={validationFailed.offlineToken}
+                   validated={validationStateToken}>
+            <TextArea id='offline-token'
```

Comment:
So, are offline tokens really multi-line things? Are these PEM certificates?

Reply:
They are quite huge. Their length varies, but in most cases it's 600+ characters long. You can try to generate yours here: https://access.redhat.com/management/api

Reply:
Urgh! Well, nothing we can change on our side about that :cry: